src/arch/arm/config.cmake

```diff
@@ -20,6 +20,10 @@ elseif(KernelArmCortexA53)
     set(KernelArmICacheVIPT ON)
     set(KernelArmPASizeBits40 ON)
     math(EXPR KernelPaddrUserTop "(1 << 40)")
+elseif(KernelArmCortexA55)
+    # set(KernelArmICacheVIPT ON)
```

Comment:
Why is this commented out?

Reply:
Not clear on whether it is needed. The manual states the D-Cache is VIPT but doesn't mention anything about the I-Cache.

Reply:
file:///home/axehei01/Downloads/temp/cortex_a55_trm_100442_0200_00_en.pdf at "B1.27 CTR, Cache Type Register" says 
```
L1Ip, [15:14]
Instruction cache policy. Indicates the indexing and tagging policy for the L1 Instruction cache:
0b10 Virtually Indexed Physically Tagged (VIPT). 
```

Reply:
Ty, uncommented.